test/java/seedu/addressbook/util/TypicalTestPersons.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.util;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.UniquePersonList;
+
+/**
+ * Class to generate typical test persons
+ */
+public class TypicalTestPersons {
+
+    public static TestPerson amy, bill, candy, donald, grace;
+
+    public TypicalTestPersons() {
+        try {
+            amy = new PersonBuilder().withName("Amy Buck").withPublicPhone("91234567")
+                    .withPublicEmail("cs@gmail.com").withPublicAddress("1 Clementi Road").build();
```

Comment:
telephone `91119111` and email `ab@gmail.com` instead of random values? (for the same reasons as before; if we follow patterns, it's easy to detect when things go wrong)